### DISCUSSION TITLE
How fast is minio's theoretical speed in this case ?

### DISCUSSION THREAD

**Discussion by edisoncoc** [minio/minio#12436] [Q&A]
I want to know how fast minio's read/write speed theory will be in this case.

The cluster I built now has a downlink speed of only 800MB/s.

I want to know how to increase the speed of my minio cluster?
## My Environment
I use 4 servers, each with 2 nodes, each node has 16 disks, a total of 128 disks.
Each of servers has a 10Gbps network card.
Every hard drive is HDD.

●  8.10.163.37:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.38:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.41:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.42:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.51:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.52:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.59:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

●  8.10.163.58:9000
   Uptime: 12 hours
   Version: 2021-04-22T15:44:28Z
   Network: 8/8 OK
   Drives: 16/16 OK

**Comment by harshavardhana** [comment]
For production setups, we suggest that you get a subscription - that comes with performance tweaks, and architecture help https://minio.io/pricing

Other than that you may ask these questions at https://slack.min.io ..
